README.md

```diff
@@ -1,70 +1,52 @@
-# Azure SDK for Javascript
+# Azure SDK for JavaScript
+
 
 | Component            | Build Status                                                                                                                                                                                |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Management Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/138?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=138&branchName=master) |
 | Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/45?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=45&branchName=master)   |
 
-This project provides an isomorphic Javascript package with TypeScript definitions that makes it easy to consume and manage
-Microsoft Azure Services.
-It supports SDKs for:
-
-- ARM services (control plane) (packages with the naming convention of `@azure/arm-<servicename>`)
-- data plane of some Azure services (packages with the naming convention of `@azure/<servicename>`).
```

Comment:
I really liked the above tip on understanding the "kinds" of libraries in this repo 
Can we keep it or an improved version of it?

Reply:
I thought the descriptions below was better! Do you disagree/want this in addition to that?

Reply:
A single line statement that tells the user the difference between management and data plane at an earlier section of the README would still be helpful

Reply:
I agree. Given the previous text wasn't really helpful in this regard either, let's tackle this with subsequent updates.